sample/src/main/res/layout/activity_main.xml

```diff
@@ -53,35 +42,49 @@
             android:id="@+id/dracula"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/item_margin_horizontal"
             android:layout_marginTop="16dp"
+            android:layout_marginRight="@dimen/item_margin_horizontal"
             android:minHeight="56dp"
             android:text="Dracula"
             android:textAllCaps="false"
             android:textColor="@android:color/white"
             android:theme="@style/DraculaOverlay" />
 
         <Button
-            android:id="@+id/custom"
+            android:id="@+id/only_gif"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/item_margin_horizontal"
             android:layout_marginTop="16dp"
+            android:layout_marginRight="@dimen/item_margin_horizontal"
+            android:layout_marginBottom="32dp"
             android:minHeight="56dp"
-            android:text="Custom"
+            android:text="OnlyGif"
             android:textAllCaps="false"
+            android:textColor="@android:color/white"
+            android:theme="@style/ZhihuOverlay" />
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="56dp"
+            android:gravity="center_vertical"
+            android:paddingLeft="@dimen/item_margin_horizontal"
+            android:text="@string/app_name"
             android:textColor="@android:color/black"
-            android:theme="@style/CustomOverlay" />
+            android:textSize="22sp" />
```

Comment:
Is this dynamic to different screen size or we may use @kundarmah 's implementation to cover?

Reply:
This is only for the sample app, changing this won't have any changes in our app. Just had to change this to avoid build errors